sources/imdsclient/src/lib.rs

```diff
@@ -157,6 +157,12 @@ impl ImdsClient {
         self.fetch_string(&instance_type_target).await
     }
 
+    /// Gets the instance-id from instance metadata.
+    pub async fn fetch_instance_id(&mut self) -> Result<Option<String>> {
+        let instance_type_target = "meta-data/instance-id";
```

Comment:
Nit: Another potential candidate for a constant

Reply:
Looking at how the other `fetch...` functions are structured in this library, I would like to stick to the `imdsclient` convention here and just keep it the same.